karyon3-log4j2/src/main/java/com/netflix/karyon/log4j/ArchaiusLog4J2ConfigurationModule.java

```diff
@@ -16,5 +28,35 @@
     @Override
     protected void configure() {
         this.requestStaticInjection(ArchaiusLog4J2ConfigurationFactory.class);
+        
+        Multibinder<Log4jConfiguration> appenderBinder = Multibinder.newSetBinder(binder(),
+				Log4jConfiguration.class);
+		appenderBinder.addBinding().to(ConsoleAppenderConfiguration.class);
+
+    }
+    
+    public static class ConsoleAppenderConfiguration implements Log4jConfiguration
```

Comment:
I will break out to another file... Can I do config with properties in another pull request, as an enhancement to the Console Appender?